### PR TITLE
add repo link, add 'Edit this page' link

### PR DESCRIPTION
**Pull request by abdullahtariq1171**
Added repo link in header and 'Edit this page' link to engage and help people who want to contribute.

### CODE REVIEW

**Comment by abdullahtariq1171** [comment]
image

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ module.exports = {
   base: '/',
   title: "Nanoutils",
   themeConfig: {
+    repo: 'nanoutils/nanoutils',
```
Comment (Beraliv) [inline docs/.vuepress/config.js]: That's fine

**Comment by Beraliv** [comment]
@abdullahtariq1171 what about this PR?

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ module.exports = {
...
+    editLinks: true,
```
[the remark this answers is not in the document]
Comment (abdullahtariq1171) [reply docs/.vuepress/config.js]: aah i see, agree with you.
Updating in a minute.

**Pull request merged**